karma.mc.config.js

```diff
@@ -143,7 +143,8 @@ module.exports = function(config) {
               path.resolve("lib/ASRouterTargeting.jsm"),
               path.resolve("lib/ASRouterTriggerListeners.jsm"),
               path.resolve("lib/OnboardingMessageProvider.jsm"),
-              path.resolve("lib/CFRMessageProvider.jsm")
+              path.resolve("lib/CFRMessageProvider.jsm"),
+              path.resolve("lib/CFRPageActions.jsm")
```

Comment:
Outstanding issue for tests: https://bugzilla.mozilla.org/show_bug.cgi?id=1486617